spec/Takeaway_spec.rb

```diff
@@ -4,16 +4,10 @@
 describe Takeaway do
   subject(:takeaway) { described_class.new }
 
-  describe '#read_menu' do
-    it "Shows the menu with the prices of each dish" do
-      expect(subject.read_menu).to eq(Menu::MENU)
```

Comment:
because you declared subject(:takeaway) above you can write expect(takeaway.read_menu)
It doesnt change anything but I think makes the code more readable, you might get confused on subject when you see it alot, but you know exactly what it is testing if you put takeaway